README.md

```diff
@@ -21,8 +21,10 @@ Framework](https://dotnet.microsoft.com/download/dotnet-framework) except for
 
 ## Getting Started
 
-If you are new here, please see [get started in 5
-minutes](./docs/trace/getting-started/README.md).
+If you are new here, please see getting started doc for
+[logs](./docs/logs/getting-started/README.md),
+[metrics](./docs/metrics/getting-started.md) and
+[trace](./docs/trace/getting-started/README.md).
```

Comment:
`logs, metrics, and trace` seems like it should be `logs, metrics, and tracing` but not sure if you are matching up to spec or something?

Reply:
Our namespaces and folder names are all `Trace` not tracing, so I sticked with that.